noetic/distribution.yaml

```
@@ -6602,6 +6602,22 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/robotis_manipulator.git
       version: noetic-devel
     status: maintained
+  rom_motor_msgs:
+    doc:
+      type: git
+      url: https://github.com/GreenGhostMan/rom_motor_msgs-doc.git
```

@GreenGhostMan hmm this repository is mostly empty.

---

```suggestion
      url: https://github.com/GreenGhostMan/rom_msgs.git
```

The doc url is typically the standard source url. Enabling it here just tells the buildfarm to run the documentation job. It's very rare to want to document a different repository. Sometimes there's a preference for a different branch.